dev/refresh_source.py

```diff
@@ -434,7 +434,8 @@ def write_deck_run_script(self, repository):
 cd "$d"
 LOG_DIR=${{LOG_DIR:-../logs}}
 
-bash -c "(./start.sh >> '$LOG_DIR/{name}.log') 2>&1\
+npm install >& "$LOG_DIR/deck.log"
```

Comment:
`start.sh` tries to set the right node version based on Deck's `package.json`, but makes a wrong assumption (for dev vms) about where `nvm` is located. Even if it doesn't find `nvm`, it runs `npm run start-dev-server`. I can keep `start.sh` or get rid of it or fix it, it's just logging a confusing warning now.

Matt, I'm not seeing either of the `{name}` log files. How are you generating Deck's `start_dev.sh`?